ocs_ci/ocs/cosbench.py

```diff
@@ -590,3 +601,42 @@ def cosbench_teardown(self):
         self.cosbench_config.delete()
         self.ns_obj.delete_project(self.namespace)
         self.ns_obj.wait_for_delete(resource_name=self.namespace, timeout=90)
+
+    def get_performance_result(self, workload_name, workload_id, size):
+        workload_file = self.get_result_csv(
+            workload_id=workload_id, workload_name=workload_name
+        )
+        throughput_data = {}
+        bandwidth_data = {}
+        with open(workload_file, "r") as file:
+            reader = csv.reader(file)
+            header = next(reader)
+            if header is not None:
+                for row in reader:
+                    op_type = row[1]
+                    throughput = row[13]
+                    bandwidth = row[14]
+                    throughput_data[op_type] = throughput
+                    bandwidth_data[op_type] = bandwidth
```

Comment:
you don't need the op_yype, throughput and bandwidth variables.
it can be :
```
throughput_data[row[1]] = row[13]
bandwidth_data[row[1]] = row[14]
```

Reply:
Fixed.